pkgs/by-name/sn/snander/package.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  makeWrapper,
+  libusb1,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "snander";
+  version = "1.7.8-unstable-2025-03-28";
+
+  src = fetchFromGitHub {
+    owner = "Droid-MAX";
+    repo = "SNANDer";
+    rev = "5810c63";
+    hash = "sha256-YsNWjlbUTAURD6eOt8ZVDBGMZT4w0hGFO6N42CdX2XA=";
+  };
+
+  buildInputs = [ libusb1 ];
+
+  buildPhase = ''
+    make -C src
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp src/snander $out/bin/
+  '';
+
+  meta = with lib; {
+    description = "Serial Nor/nAND/Eeprom programmeR (based on CH341A)";
+    homepage = "https://github.com/Droid-MAX/SNANDer";
+    license = licenses.gpl2;
```

The README explicitly states “GNU GPL, version 2.0, or later”.

```suggestion
    license = lib.licenses.gpl2Plus;
```

---

Thank you for pointing this out. I will have it changed in my commit.